api/lib/block-listeners/base-node-subscription.js

```diff
@@ -174,7 +174,8 @@ class BaseNodeSubscription {
       // transaction to each subscribed user.
       // TODO doesn't handle failing txs as it doesn't extract addresses from those txs (they are not tagged)
       block.transactions.forEach((tx) => {
-        tx.involvedAddresses.forEach((address) => {
```

Comment:
what is the difference between `involvedAddresses` vs. `transactionMessageAddresses`?

Reply:
where is this "published"?

https://github.com/luniehq/lunie/pull/5076/files#diff-0ad2d28223c295dd602e00c42eab16aeR187

Reply:
in Redis. This is the subscription that `listenToNotifications` listens to, in line 31 in `notificationController.js`

Reply:
> what is the difference between involvedAddresses vs. transactionMessageAddresses?

The difference is that involvedAddresses contains all the addresses involved per "transaction", whereas `transactionMessageAddresses` are the addresses involved per message